File: plusMinus/plus_minus.py
import math
import os
import random
import re
import sys

arr = [-4, 3, -9, 0, 4, 1]

def plus_minus(arr):
    positive = 0
    negative = 0
    zero = 0

    for i in arr:
        if i > 0:
            positive +=1
        if i < 0:
            negative +=1
        if i == 0:
            zero +=1

    positiveRatio = positive/len(arr)
    negativeRatio = negative/len(arr)
    zeroRatio = zero/len(arr)

    print("{:.6f}".format(positiveRatio))
    print("{:.6f}".format(negativeRatio))
    print("{:.6f}".format(zeroRatio))




plus_minus(arr)

File: plusMinus/plusminus.js
const arr = [-4, 3, -9, 0, 4, 1]
const arrTwo = [-2]

function plusMinus(arr) {
    let positive = 0
    let negative = 0
    let zero = 0

    for (let index = 0; index < arr.length; index++) {
        const element = arr[index];
        if (element < 0) {
            negative++
        }
        if ( element > 0){
            positive++
        }
        if (element === 0) {
            zero++
        }
    }
    let positiveRatio = (positive/(arr.length)).toFixed(6)
    let negativeRatio = (negative/(arr.length)).toFixed(6)
    let zeroRatio = (zero/(arr.length)).toFixed(6)

    console.log(positiveRatio)
    console.log(negativeRatio)
    console.log(zeroRatio)
    
}

plusMinus(arr);
